internal/ingress/controller/kong.go

```diff
@@ -100,13 +103,3 @@ func (n *KongController) fetchCustomEntities() ([]byte, error) {
 	}
 	return config, nil
 }
-
-// getIngressControllerTags returns a tag to use if the current
-// Kong entity supports tagging.
-func (n *KongController) getIngressControllerTags() []string {
-	var res []string
-	if n.cfg.Kong.HasTagSupport {
-		res = append(res, n.cfg.Kong.FilterTags...)
-	}
-	return res
-}
```

Comment:
`getIngressControllerTags` is only used by KIC 1.x (and is a part of a mechanism deprecated in 2.x), so I think there's no value in moving it to `package sendconfig`, however there's a cost - `package sendconfig` being burdened by a 1.x-specific enabling mechanism that will be phased out very soon.

Therefore, please move this function implementation back to this file.

Rule of thumb: "don't scatter deprecated concepts around the codebase, keep them as isolated as possible"

Reply:
sure! got it 👍🏻

Reply:
Oh. I realized that would save us a round trip if I did it right away, and I've pushed a commit performing this already :sweat_smile: